src/main/java/org/sonarlint/intellij/util/ConfigurationSharing.kt

```diff
@@ -93,6 +93,12 @@ class ConfigurationSharing {
                         )
                         SonarLintConsole.get(project).error("Error while creating the shared file, IO exception : " + e.message)
                     }
+
+                    get(project).simpleNotification(
+                        null,
+                        "File \"$filename\" has been created. It might take some time for changes to be reflected in the project view.",
```

Comment:
```suggestion
                        "File '$filename' has been created. It might take some time for changes to be reflected in the project view.",
```
Based on IntelliJ guidelines, we should not put double quotes in sentences